Source/externs.m

```diff
@@ -267,22 +267,50 @@
 
 // Pasteboard Type Globals
 NSString *NSStringPboardType = @"NSStringPboardType";
+NSString *NSPasteboardTypeString = @"NSStringPboardType";
+
 NSString *NSColorPboardType = @"NSColorPboardType";
-NSString *NSFileContentsPboardType = @"NSFileContentsPboardType";
-NSString *NSFilenamesPboardType = @"NSFilenamesPboardType";
+NSString *NSPasteboardTypeColor = @"NSColorPboardType";
```

Comment:
I think it is better to assign the first definition to the second. Although I hope the compiler will try to do the same. That way we are on the save side if somebody tries to compare the constant strings as pointers. (Yes, this happens in application code)